pipeline/transform/matches.py

```diff
@@ -24,7 +30,10 @@ def convert_matches_from_df(df: pd.DataFrame) -> List[Match]:
     cols = set(df.columns)
     field_names = [f.name for f in dataclasses.fields(Match) if f.name in cols]
     user_dicts = df[field_names].to_dict(orient="records")
```

Comment:
This variable can be renamed to `match_dicts`, it was my mistake that it was called `user_dicts` because I just copied the code for the user transform :)